libs/styles/src/lib/scss/components/_toast.scss

```diff
@@ -2,7 +2,8 @@
   --comment: 'A toast aka floating notification';
 
   position: relative;
-  width: 300px;
+  min-width: 300px;
+  width: min-content;
```

Comment:
When a word is too long to fit on screen, it might go out of viewport boundaries (relevant for mobile screens): 

![image](https://github.com/e-gov/cvi/assets/1833029/664a41b7-47e2-41a8-b160-8756477f6292)

It could be useful to add `word-break: break-word; overflow-wrap: break-word;` to `__message` to prevent this and break those overly long words.